src/wodan/wodan_map_407.ml

```diff
@@ -1,513 +0,0 @@
-(**************************************************************************)
```

Comment:
I don't want to drop 4.06 compatibility. Please keep this.

Reply:
We don't need this to keep 4.06 compatibility, I think the `open Stdcompat` in `keyedmap.ml` does the job, and the build is passing in the CI with 4.06.

Reply:
Okay then.